src/renderer/lib/utils.ts

```diff
@@ -61,3 +62,12 @@ export const humanReadableBytes = (bytes: number): string => {
 
   return `0 ${sizes[0]}`;
 };
+
+export const isSubdir = (parentPath: string, dirPath: string) => {
```

Comment:
can we rename this to `isSubdirectory` to avoid short names in public exported functions and interfaces?